src/query/scan_manager.c

```diff
@@ -7879,16 +7880,27 @@ scan_print_stats_text (FILE * fp, SCAN_ID * scan_id)
       break;
     }
 
-  fprintf (fp, " time: %d, fetch: %llu, ioread: %llu", TO_MSEC (scan_id->scan_stats.elapsed_scan),
+  /* need space for the followed "time ... " except for aggregate optimization */
+  if (scan_id->scan_stats.agg_index_name == NULL)
+    {
+      fprintf (fp, " ");
+    }
+
+  fprintf (fp, "time: %d, fetch: %llu, ioread: %llu", TO_MSEC (scan_id->scan_stats.elapsed_scan),
 	   (unsigned long long int) scan_id->scan_stats.num_fetches,
 	   (unsigned long long int) scan_id->scan_stats.num_ioreads);
 
   switch (scan_id->type)
     {
     case S_HEAP_SCAN:
     case S_LIST_SCAN:
-      fprintf (fp, ", readrows: %llu, rows: %llu)", (unsigned long long int) scan_id->scan_stats.read_rows,
+      fprintf (fp, ", readrows: %llu, rows: %llu", (unsigned long long int) scan_id->scan_stats.read_rows,
 	       (unsigned long long int) scan_id->scan_stats.qualified_rows);
+      if (scan_id->scan_stats.agg_index_name)
+	{
+	  fprintf (fp, ", agl=%s", scan_id->scan_stats.agg_index_name);
```

Comment:
I think it is consistent and natural for the index name to come immediately after SCAN, as shown below.
```
SCAN (index: participant.fk_participant_host_year), (btree time: 1, ...) (lookup time: 0, rows: 14)
```
And the output should also use ':', not '='.
```
SCAN (agl: dba.t1.pk_t1_id), (btree time: 1, ...) (lookup time: 0, rows: 14)
```

Reply:
btree is not a scan method in this case because the it just provides statistics in its header. Therefore, btree-printed should be excluded as a scan method. How about "SCAN (table: dba.t1), (time: 0, fetch: 0, ioread: 0, readrows: 0, rows: 0, agl: pk_t1_id)" ?

Reply:
Is there any way below?
```
SCAN (table: dba.t1, agl=pk_t1_id), (time: 0, fetch: 0, ioread: 0, readrows: 0, rows: 0)
```

Reply:
At least we should use '=' instead of ':'.